component/file_cache/file_cache.go

```diff
@@ -98,7 +98,7 @@ type FileCacheOptions struct {
 	// v1 support
 	V1Timeout     uint32 `config:"file-cache-timeout-in-seconds" yaml:"-"`
 	EmptyDirCheck bool   `config:"empty-dir-check" yaml:"-"`
-	SyncToFlush   bool   `config:"sync-to-flush" yaml:"sync-to-flush,omitempty"`
```

Comment:
I believe that this should keep the omitempty flag.

Reply:
I'm not 100% on this, but I looked into it a bit, and I think `omitempty` means that if this flag is false, then it would be omitted when writing the struct out to YAML.
So I figured that if it's true by default, then if it's false, it needs to be written to the YAML. Because if it isn't, then it will be set true by default when it's read back in.

Reply:
Oh then yeah I think you are right.